File: HalfWerk.FeWebshop/src/app/models/bestelling.ts
import { ContactInfo } from './contactInfo';
import { Afleveradres } from './afleveradres';
import { BestelRegel } from './bestelregel';
import { BestelStatus } from '@models/bestelStatus';

export class Bestelling {
  constructor() {}

  id: number;
  factuurnummer: number;
  klantnummer: number;
  contactInfo: ContactInfo;
  afleverAdres: Afleveradres;
  bestelRegels: BestelRegel[];
  besteldatum: string;
  bestelStatus: BestelStatus;
  factuurTotaalExclBtw: number;
  factuurTotaalInclBtw: number;
}

File: HalfWerk.CommonModels/DsBestelService/Models/Bestelling.cs
using System;
using System.Collections.Generic;

namespace HalfWerk.CommonModels.DsBestelService.Models
{
    public class Bestelling
    {
        public long Id { get; set; }
        public long Factuurnummer { get; set; }
        public long Klantnummer { get; set; }
        public ContactInfo ContactInfo { get; set; }
        public Afleveradres Afleveradres { get; set; }
        public BestelStatus BestelStatus { get; set; }      
        public DateTime Besteldatum { get; set; }
        public decimal FactuurTotaalExclBtw { get; set; }
        public decimal FactuurTotaalInclBtw { get; set; }

        public List<BestelRegel> BestelRegels { get; set; }
    }
}
